src/persist-client/src/impl/state.rs

```diff
@@ -78,21 +78,18 @@ where
         read_cap
     }
 
+    // NOTE(aljoscha): This never yields an `InvalidUsage` error now.
```

Comment:
it does if the writer is expired, no?

Reply:
ah, I was overeager and missed the `?` on `self.writer(writer_id)?`... 🙈

Reply:
ah, I was overeager and missed the `?` on `self.writer(writer_id)?`... 🙈